images/kubeadm/runner

```diff
@@ -21,14 +21,6 @@ if [ ! -e test-infra ]; then
   git clone https://github.com/kubernetes/test-infra
```

Comment:
should this be moved to `--repo` as well eventually?

Reply:
this should be left up to the underlying runner in the base image, when using bootstrap.py we have a runner that does some things like that. that runner is also needed for EG the bazel cache.

Reply:
really we should just do away with this runner being the entrypoint and do any of:
- do things ahead of time, eg pre-installing tools in the image
- let bootstrap check things out
- let the testing script do things
instead of the runner / entrypoint.

Reply:
i need to look up what the base image is.